brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3436 Fix: Global - Resolved an issue where adding containers, forms, popup builder and lottie caused errors when the gutenberg plugin was active.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3436

### Screenshots

- NA

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Activate latest version of gutenberg.
- Drag and drop Container, Lottie, Forms and Visit Popup builder.
- Make sure non of our block shows “This block has encountered an error” issue

### Checklist:
